cmd/update/project.go

```diff
@@ -80,7 +81,7 @@ func updateProjectsFunc(ctx context.Context, args []string, cmdCtx cmdCore.Comma
 	archiveProject := projectConfig.ArchiveProject
 	activateProject := projectConfig.ActivateProject
 	if activateProject == archiveProject {
-		fmt.Printf(errInvalidUpdate)
+		fmt.Println(errInvalidUpdate)
```

Comment:
Instead of printing , can we do the same here aswell , return the error and let the main.go handle it by logging it